code/game/objects/items/weapons/storage/lunchbox.dm

```diff
@@ -359,96 +349,253 @@
-	icon_state = "lunchbox_syndie"
-	item_state = "toolbox_red"
-	force = 10
-	throwforce = 6
-
-/obj/item/weapon/storage/lunchbox/syndie/pre_filled
+/obj/item/weapon/storage/lunchbox/plastic/nt/pre_filled
 	has_lunch = TRUE
 
-/obj/item/weapon/storage/lunchbox/syndie/New()
-	..()
-	if(has_lunch == TRUE)
-		pickfood_syndie()
-
 // Getmore themed lunchbox
-/obj/item/weapon/storage/lunchbox/nt/getmore
+/obj/item/weapon/storage/lunchbox/plastic/nt/getmore
 	name = "Getmore lunchbox"
 	desc = "A little metal lunchbox. This one has the Getmore Chocolate Corp logo printed on the side."
 	icon_state = "lunchbox_getmore"
 	item_state = "toolbox_blue"
 
-/obj/item/weapon/storage/lunchbox/nt/getmore/pre_filled
+/obj/item/weapon/storage/lunchbox/plastic/nt/getmore/pre_filled
 	has_lunch = TRUE
 
-// Discount Dan themed lunchbox
-/obj/item/weapon/storage/lunchbox/discount
-	name = "Discount lunchbox"
-	desc = "A little cardboard lunchbox. This one has the Discount Dan logo printed on the side. It looks very flimsy, and has a musty smell even when empty."
-	icon_state = "lunchbox_discount"
-	item_state = "toolbox_purple"
-	force = 2
-	hitsound = 'sound/weapons/tap.ogg'
-	attack_verb = list("taps", "smacks")
-	throwforce = 1
-	starting_materials = list(MAT_CARDBOARD = 3750)
-	w_type = RECYK_MISC
+// Randomized collectable lunchboxes
+/obj/item/weapon/storage/lunchbox/plastic/nt/random
+	name = "Collectible lunchbox"
+	desc = "A plastic lunchbox with a unique design!"
+	icon_state = "lunchbox_random"
 
-/obj/item/weapon/storage/lunchbox/discount/pre_filled
+/obj/item/weapon/storage/lunchbox/plastic/nt/random/New()
+	..()
+	if(has_lunch == TRUE)
+		pickfood()
+	switch(rand(1,14))
+		if(1)
+			name = "Ian lunchbox"
+			desc = "A little plastic lunchbox. This one has a portrait of Ian on the side."
+			icon_state = "lunchbox_ian"
+			item_state = "toolbox_orange"
+		if(2)
+			name = "Shard lunchbox"
+			desc = "A little plastic lunchbox. This one has a supermatter shard on the side, and a red line across a hand reaching out to touch it."
+			icon_state = "lunchbox_shard"
+			item_state = "toolbox_lightblue2"
+		if(2)
+			name = "AI lunchbox"
+			desc = "A little plastic lunchbox. This one has a design on the side depicting a station's AI."
+			icon_state = "lunchbox_ai"
+			item_state = "toolbox_grey"
+		if(3)
+			name = "Beepsky lunchbox"
+			desc = "A little plastic lunchbox. This one has a picture of Beepsky on the side, and bright red letters spelling out 'LAW'."
+			icon_state = "lunchbox_beepksky"
+			item_state = "toolbox_red"
+		if(4)
+			name = "Carp lunchbox"
+			desc = "A little plastic lunchbox. This one has a picture of a space carp on the side."
+			icon_state = "lunchbox_carp"
+			item_state = "toolbox_purple"
+		if(5)
+			name = "Carp lunchbox"
+			desc = "A little plastic lunchbox. This one has a picture of a space carp on the side."
+			icon_state = "lunchbox_carp"
+			item_state = "toolbox_purple"
```

Comment:
Duplicate carp lunchbox

Reply:
Same thing with the jannie lunchbox

Reply:
Thanks for catching that. Should be fixed now.